app/controllers/external_users/claim_types_controller.rb

```diff
@@ -1,53 +1,51 @@
 class ExternalUsers::ClaimTypesController < ExternalUsers::ApplicationController
   skip_load_and_authorize_resource
 
-  before_action :init_claim_types
-  before_action :set_claim_types_for_provider, only: [:selection]
+  before_action :set_available_fee_schemes_for_provider, only: %i[selection]
+  before_action :disable_primary_navigation, only: %i[selection]
```

Comment:
🗒  In the new design, the navigation is not shown for this page. Every other page still needs to show the menu. This is not an ideal solution IMO but that's the one that I came up with to avoid embracing a major refactor of the views just to achieve this. Hopefully this will be removed entirely once all the other pages also get updated.